components/gallery/useGalleryItem.ts

```diff
@@ -41,6 +41,7 @@ export const useGalleryItem = () => {
   const nftMetadata = ref<NFTWithMetadata>()
 
   const { params } = useRoute()
+  params.item = nftId || params.item
```

Comment:
imo it's not wise to mutate url params but 

may @roiLeo or @preschian would have opinion

Reply:
![pather](https://user-images.githubusercontent.com/9987732/233361193-923914f6-55d5-4446-b4e6-ebc9a7ed2029.jpg)